spec/models/contact_spec.rb

```diff
@@ -4,7 +4,21 @@
 # rubocop:disable RSpec/EmptyExampleGroup
 RSpec.describe Contact, type: :model do
   context "with validations" do
-    subject { FactoryBot.build(:contact) }
+    subject(:contact) { FactoryBot.build(:contact) }
+
+    it { is_expected.to validate_presence_of(:address) }
+
+    # rubocop:disable RSpec/MultipleExpectations
+    it { is_expected.to validate_presence_of(:birth_date) }
+
+    it "is expected to validate that :birth_date is ISO8601" do
+      contact.birth_date = "01 Jan 1970"
+      expect(contact).to be_invalid
+
+      contact.birth_date = "19700101"
+      expect(contact).to be_valid
```

Comment:
When testing validator classes, I sometimes prefer to write a separate test file for it by defining a throwaway active model using the validator on the test file and making assertions over it.